apps/app/utils/pushNotification.ts

```
@@ -118,6 +124,10 @@ export const initializePushNotification = async () => {
 export const setupMessagingListeners = () => {
   const unsubscribeOnMessage = onMessage(messaging, async remoteMessage => {
     console.log('[FCM] Foreground message:', remoteMessage.messageId);
+    const unreadCount = remoteMessage.data?.unreadCount;
+    if (unreadCount !== undefined) {
```

ESLint

[error] Unexpected use of undefined.

(no-undefined)